UKHomeOffice/sgmr-service
# SPCR-405 Update site maintenance page

### AC / Description of changes
This PR implements the site maintance page content changes proposed in [SPCR-405](https://viabledata.atlassian.net/browse/SPCR-405).

### To Test
* Run linter and unit tests
* All should pass
* set `siteMaintenance` to `false` in `config.js`
* `npm start`
* You should see the site maintenance page
* It should contain the content described in the ticket

### Notes

---

**Remember to merge to feature branches not `master`.**
